src/backend/app/config.py

```diff
@@ -20,7 +21,7 @@ class Settings(BaseSettings):
     DEBUG: bool = False
     LOG_LEVEL: str = "INFO"
 
-    EXTRA_CORS_ORIGINS: Optional[Union[str, list[str]]] = []
+    EXTRA_CORS_ORIGINS: Optional[Union[str, list[str]]] = os.environ.get("EXTRA_CORS_ORIGINS","").split(",")
```

Comment:
We don't need to accept the environment like this. It does look at the EXTRA_CORS_ORIGINS by itself.

Reply:
Thanks, Didn't knew it was possible with `pydantic-settings`. I have updated the code!